backend/apps/vacancies/admin.py

```diff
@@ -4,11 +4,41 @@
 from .models import Currency, Vacancy
 
 
+@admin.action(description="Mark selected vacancies as active")
+def activate(self, request, queryset):
+    for vacancy in queryset:
+        vacancy.is_active = True
+        vacancy.save()
+
+
+@admin.action(description="Mark selected vacancies as deactive")
```

Comment:
Mark selected vacancies as inactive / Deactivate the selected vacancies

Reply:
fixed